framework/sandstone.h

```diff
@@ -341,6 +341,10 @@ typedef enum test_flag {
     /// Tells the --test-tests mode to ignore the detection that this test
     /// may have called test_time_condition() before doing any work.
     test_flag_ignore_do_while       = 0x0100,
+
+    /// Indicates that a test can only attribute failure to a particular
+    /// package and not to threads or cores.
+    test_failure_package_only       = 0x1000,
```

Comment:
I was attempting to follow the existing pattern here.  There are 3 bits free between the existing groups of flags (i.e., between test_type_kvm and test_flag_ignore_memory_use) so I left 3 bits between test_flag_ignore_do_while and test_failure_package_only.

Reply:
You could move it above the test-test mode numbers (0x02, 0x04 and 0x08 are free).

But since we don't need to keep binary compatibility, we can always reshuffle if we want to.